### DISCUSSION TITLE
Why is the Flutter linter still disabled in the boilerplate ?

### DISCUSSION THREAD

**Discussion by lepresk** [nylo-core/nylo#168] [General]
A few months ago, I submitted a PR to enable the Flutter linter in this boilerplate, which I believe is essential for maintaining code standards, formatting, and catching errors. However, after creating a new project yesterday, I noticed that the linter is disabled again.

My question is : why is the linter deactivated in 2024, especially when it's widely considered best practice to use it in modern projects? It seems counterproductive, and I'd love to understand the reasoning behind this decision.

**Comment by agordn52** [comment]
Hey @lepresk,

Hope you are well!
Got some exciting news on v6 coming soon, if you wanted an early look at the features I'd be happy to arrange a call to show you what's new, just let me know. The release date will be 1st October.

Re. your linter PR
It's still in the default setup for new projects, it's just not enabled by default, [see](https://github.com/nylo-core/nylo/blob/c7846b3183d4b008538477d794782f2dd6ea9cea/pubspec.yaml#L35).

My decision to not enable it by default is because most (beginner) Flutter developers don't understand 'coding standards'. Personally, I use the linter package in my projects and agree with your points. I think it's best to make Nylo accessible for all developers on setup but for intermediate/senior devs, they have the option to quickly opted in to the linter by uncommenting that [line](https://github.com/nylo-core/nylo/blob/c7846b3183d4b008538477d794782f2dd6ea9cea/pubspec.yaml#L37).

If you disagree, let me know too. I think it's good to have these kinda debates to make it better for everyone. Hope you can understand my thinking on this.

Fyi - Nylo's packages going forward in v6 are using the [Linter](https://pub.dev/packages/flutter_lints), e.g. nylo_support and nylo_framework

**Reply by lepresk** [reply]
I think lintter should be enabled by default to help beginners learn how to write good code

**Comment by lepresk** [comment]
Hi @agordn52 for the v6, I have planned a change that will be a breaking change. It is related to the API helper, which is not currently type-safe and returns a dynamic type. We can certainly discuss all these details if you would like
